auraed/src/runtime/pod_service/pod_service.rs

```diff
@@ -60,14 +67,44 @@ impl pod_service_server::PodService for PodService {
         &self,
         request: Request<PodServiceAllocateRequest>,
     ) -> Result<Response<PodServiceAllocateResponse>, Status> {
-        let _request = request.into_inner();
+        let request = request.into_inner();
+        let pod = request.pod.expect("pod");
+        let name = pod.name;
+        let image = pod.image;
+        info!(
+            "PodService: allocate() name={:?} image={:?}",
+            name.clone(),
+            image.clone()
+        );
+
+        let _container_bundle = bundle::container::container(&image);
```

Comment:
@future-highway here is where we need to turn an "image string" into an "OCI bundle" on the local filesystem. 

After the OCI bundle is "decompressed" locally we can then use the files in the bundle to build the container below.